test/common_test.go

```diff
@@ -319,6 +319,18 @@ func (c *T) RecvStart() bool {
 		return false
 	}
 	defer m.Free()
+	if addr := m.Port.Address(); addr != c.addr {
+		c.Debugf("Got unexpected message port address: %s", addr)
```

Comment:
Since you're returning false, make these Errorf().  It will be easier to debug the CI failures his way.